R/assertions.R

```diff
@@ -1424,3 +1424,51 @@ assert_date_var <- function(dataset, var, dataset_name = NULL, var_name = NULL)
     ))
   }
 }
+
+#' Is a Variable is a Date or Datetime Variable?
+#'
+#' Checks if a variable vector is a date or datetime variable
+#'
+#' @param var The variable to check
+#'
+#' @param var_name The name of the variable. If the argument is specified, the
+#'   specified name is displayed in the error message.
+#'
+#' @return
+#' The function throws an error if `var` is not a date or datetime variable
+#' and returns the input invisibly otherwise.
+#'
+#' @export
+#'
+#' @author Sadchla Mascary
+#'
+#' @keywords assertion
+#'
+#' @examples
+#' library(tibble)
+#' library(rlang)
```

Comment:
You use neither of these packages so please remove these `library()` calls.

Reply:
Made this update